js/jquery/fade.js

```diff
@@ -0,0 +1,123 @@
+import $ from 'jquery'
+
+const NAME = 'fade'
+const VERSION = '1.0.0-alpha.2'
+const DATA_KEY = 'axa.fade'
+const EVENT_KEY = `.${DATA_KEY}`
+const DATA_API_KEY = '.data-api'
+const JQUERY_NO_CONFLICT = $.fn[NAME]
```

Comment:
hmh, I see.
My consideration is that we never know what kind of plugins our style-guide consumers will use and which name they will have. So I would like to stay as defensiv as possible, plus we would keep our code more DRY, because we will not need the `no-conflict code`.